drivers/dai/sof/ssp/ssp.c

```diff
@@ -0,0 +1,2143 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
```

Comment:
Any idea if we could reuse existing Zephyr I2S API? It looks there is a directory called drivers/i2s/

Reply:
Probably would be possible. But the dai API (that this ssp driver is using) is like 1 level up in generic terms, so the idea is that we can gather many other audio drivers, than just i2s, behind it. So the application using the API (like sof) can use the same API with different drivers, like dmic, hda, alh etc.